code/modules/reagents/chemistry/reagents_datum.dm

```diff
@@ -287,3 +287,6 @@
 	if(M.healthdoll)
 		M.healthdoll.cached_healthdoll_overlays.Cut()
 	M.updatehealth("fakedeath reagent end")
+
+/datum/reagent/proc/has_heart_rate_increase()
+	return heart_rate_increase
```

Comment:
do you need to do this for the overdose to increase heart rate?

Reply:
Yea, I want it to only increase on overdose